R/neha.R

```diff
@@ -7,6 +7,15 @@
 #' @return A data frame in which, in addition to all of the variables in 'eha_data', there is one column for each directed dyad, named 'i_j', where 'i' and 'j' are node ids, in which the value indicates the number of time points before 'time' that 'i' experienced the event in the respective 'cascade'. If 'i' did not experience the cascade before 'time', the value is 0.
 #' @export
 data_neha_discrete <- function(eha_data, node, time, event, cascade){
+  
+  # Ensure data is in correct format
+  if (!inherits(eha_data, "data.frame")) {
+    if (is.null(as.data.frame(eha_data))) {
+      stop("Event history data must be a data.frame object.")
+    } else {
+      eha_data <- as.data.frame(eha_data)
```

Comment:
Nit: `as.data.frame` is called twice where you could just store the result. I think a better pattern would be to use a `tryCatch`, then handle the `NULL`. That way you also get informative error messages not only if the operation returns `NULL` but also if  it throws an exception. Something like (code not tested, might have to double check the tryCatch syntax):
```
tryCatch(
    {
        eha_data <- as.data.frame(eha_data)
        stopifnot(!is.null(eha_data))
    }
    error = function(cond) {
        stop("informative error message")
   }
)
```